css/account.css

```diff
@@ -0,0 +1,83 @@
+.tabs {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  margin-top: 5rem;
+  width: 50%;
+  padding: 10px;
+}
+.tab-header {
```

Comment:
you can provide proper space in between so it could looks better, isn't it ?

Reply:
Yes.. Done